home/models.py

```diff
@@ -104,11 +105,28 @@ def __str__(self):
 
 
 class Tag(models.Model):
-    tag_name = models.CharField(max_length=50)
+    tag_name = models.CharField(max_length=20)
+    questions = models.ManyToManyField('Question', through='Question_Tag')
 
     def __str__(self):
         return self.tag_name
 
+    # Input: array of strings that represent name of tags.
+    # The function will go through the array and will add them to the DB if they don't exist yet.
+    @classmethod
```

Comment:
not sure what this funciton would be useful for - if its to let a user type-in a list of tags to be added to a question, then it would probably need to either return the tag objects, get a question object as input or be a method of Question.

I suggest you add model methods only when you have a very clear idea in your mind about where and how they would be used.